src/content/components/TimelineCanvas.js

```diff
@@ -137,12 +151,26 @@ export default class TimelineCanvas extends Component {
       hover: hoveredItem !== null,
     });
 
-    return <canvas className={className}
-                   ref='canvas'
-                   onMouseMove={this._onMouseMove}
-                   onMouseOut={this._onMouseOut}
-                   onDoubleClick={this._onDoubleClick}
-                   title={this._getHoveredItemInfo()} />;
+    const tooltipText = this._getHoveredItemInfo();
+
+    return (
+      <div>
+        <canvas className={className}
+                ref='canvas'
+                onMouseMove={this._onMouseMove}
+                onMouseOut={this._onMouseOut}
+                onDoubleClick={this._onDoubleClick} />
+        {
+          !isDragging && tooltipText
+            ? <Tooltip mouseX={mouseX}
+                       mouseY={mouseY}
+                       ref={tooltipText}>
```

Comment:
Ah, nice catch. I think I meant `key` here, and maybe this is why I resorted to some update things I didn't need to. I will investigate.